src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,49 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, onTabSelected, selectedTabId }) => {
+  const tabId = tabs.find(el => el.id === selectedTabId) === undefined
+    ? tabs[0].id
+    : selectedTabId;
```

Comment:
It looks strange, you pass the tab id to this component, so here you need to find this tab by id and print it